README.md

```diff
@@ -54,11 +54,19 @@ file\_sd\_configs can read and then then relabel as needed.
 
 ## How It Works
 
-The client registers with the proxy, and awaits instructions.
-
-When Prometheus performs a scrape via the proxy, the proxy finds
-the relevant client and tells it what to scrape. The client performs the scrape,
-sends it back to the proxy which passes it back to Prometheus.
+![Sequence diagram](./docs/sequence.svg)
```

Comment:
I'd say Network rather than NAT, as it's more general.

Reply:
Fixed :+1:

Reply:
Apologies, let me re-create that svg, I damaged something else, one moment